EDEngineer/Utils/ItemNameConverter.cs

```diff
@@ -79,7 +79,7 @@ public bool TryGet(Kind kind, string key, out string name)
 
             var formattedKey = key.ToLowerInvariant();
 
-            var entry = entryDatas.FirstOrDefault(e => e.FormattedName == formattedKey && (e.Kind & kind) == e.Kind) ??
+            var entry = entryDatas.FirstOrDefault(e => string.Compare(e.FormattedName,formattedKey, true) == 0&& (e.Kind & kind) == e.Kind) ??

```

Comment:
I think you can drop this change

instead we should add a test that ensures all FormattedName are lowercased - I can take care of this if you prefer

Reply:
Good idea, it's done.